File: JavaKotlinPracticals/app/src/main/java/com/example/javakotlinpracticals/ExceptionHandling.java
package com.example.javakotlinpracticals;

import java.util.Scanner;

public class ExceptionHandling {

    public static void main(String[] args) {

        int a, b;
        int arr[] = new int[2];

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter The Value of a:");
        a=sc.nextInt();

        System.out.println("Enter The Value of b:");
        b=sc.nextInt();

        try {
            int result=a/b;
            String str=null;
            System.out.println(str.length());
            //arr[3]=10;
        } catch(Exception e) {
            System.out.println("Exception : " +e);
        } finally {
            System.out.println("No exceptions!");
        }
    }
}

File: JavaKotlinPracticals/app/src/main/java/com/example/javakotlinpracticals/ExceptionHandling.kt
package com.example.javakotlinpracticals

import java.lang.ArithmeticException
import java.lang.Exception

fun main()
{
    //simple try catch
    try {
        var a = arrayOf(10, 20, 30, 0)
        //var c=a[0]/a[3]
        a.set(4, 50)
    } catch (e:Exception) {
        println(e)
    }

    //nested try catch
    var numerator = intArrayOf(10, 20, 30, 40, 50, 90)
    var denometer = intArrayOf(2, 0, 2, 0, 2)
    try {
        for (i in numerator.indices) {
            try {
                println(numerator[i].toString()+"/"+denometer[i]+" is:" +numerator[i] / denometer[i])
            } catch (e:ArithmeticException) {
                println("You can't divide by zero!")
            }
        }
    } catch (e:ArrayIndexOutOfBoundsException) {
        println("Element Not Found")
    }
}